thinkhazard/templates/common.jinja2

```diff
@@ -21,3 +21,9 @@
         </li>
       </ul>
 {% endmacro %}
+
+{% macro thinkhazardLogo() %}
```

Comment:
It would be better to use snake case here (ie. thinkhazard_logo). Jinja2 templates can be considered as python code and by convention python variable and function names are written in snake case.